components/form/Form.tsx

```diff
@@ -70,7 +70,7 @@ export type GetFieldDecoratorOptions = {
   exclusive?: boolean;
   /** Normalize value to form component */
   normalize?: (value: any, prevValue: any, allValues: any) => any;
-  /** Whether stop validate on first rule of error for this field.	 */
+  /** Whether stop validate on first rule of error for this field.  */
   validateFirst?: boolean;
```

Comment:
What is the difference between these two lines?

Reply:
blank spaces?

Reply:
No, I changed the two lines have the same number of spaces, but still different, now has passed, it does not matter

Reply:
No, I changed the two lines have the same number of spaces, but still different, now has passed, it does not matter